### PR TITLE
[Resource] Add bulk volumes delete

### PR DESCRIPTION
**Pull request by ismirlia**
Output from acceptance testing:

```
=== RUN   TestAccIBMPiVolumesDeleteBasic
--- PASS: TestAccIBMPiVolumesDeleteBasic (36.28s)
PASS
```

### CODE REVIEW

**Comment by ismirlia** [comment]
@hkantare @yussufsh Please review as part of power Q3 release.

File: website/docs/r/pi_volumes_delete.html.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM : ibm_pi_volumes_attach"
```
Comment (Alexander-Kita) [inline website/docs/r/pi_volumes_delete.html.markdown:4]:
```suggestion
page_title: "IBM : ibm_pi_volumes_delete"
```

File: website/docs/r/pi_volumes_delete.html.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Timeouts
+
+pi_volumes_attach provides the following [Timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) configuration options:
```
Comment (Alexander-Kita) [inline website/docs/r/pi_volumes_delete.html.markdown:40]:
```suggestion
pi_volumes_delete provides the following [Timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) configuration options:
```

Comment (ismirlia) [reply website/docs/r/pi_volumes_delete.html.markdown]: updated

**Review by michaelkad — APPROVED** [review]

**Comment by yussufsh** [comment]
This resource does not suit the Terraform IaC concepts. We are just destroying volumes during apply and subsequent apply will do nothing. I believe one main rule of Terraform is what is created by Terraform should be removed via Terraform.

Image having bulk-delete resource and a volume resource defined in the same module. They won't work in tandem leaving unexpected infrastructure state.

**Comment by ismirlia** [comment]
Opened: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/5765 closing this one

**Pull request closed without merging**